<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config</class>
 <widget class="QMainWindow" name="config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/new/prefix1/icon/favicon.ico</normaloff>:/new/prefix1/icon/favicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="js_path">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>js文件:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="js_path_found">
    <property name="geometry">
     <rect>
      <x>325</x>
      <y>10</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="deleteload_key">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="text">
     <string>重启或关闭node.js服务器时删除之前的记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="config_save">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存配置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>399</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
